cdc/cdc.cc

```diff
@@ -63,6 +64,46 @@ using namespace std::chrono_literals;
 
 static logging::logger cdc_log("cdc");
 
+cdc::options::options(const std::map<sstring, sstring>& map) {
```

Comment:
Why not place this below inside namespace cdc and drop cdc::? There's already code there. It would be more consistent to do it in one way.

Reply:
Honestly because I hate code inside namespace{}. It makes it harder to see where I am, and it also makes it uglier whenever you want to group related, but namespace separated functions (or file local statics).

Reply:
Ok. Let's then fix it the other way around and get rid of namespace cdc in .cc at all. Not in this patchset though.

Reply:
Woho!